ci-operator/step-registry/rosa/conf/machine-pool/rosa-conf-machine-pool-commands.sh

```diff
@@ -86,6 +86,7 @@ if [[ "$LOCAL_ZONE" == "true" ]]; then
     exit 1
   fi
 
+  MP_MACHINE_TYPE=$(tail -n 1 "${SHARED_DIR}/instance-types.txt")
```

Comment:
I would keep the configuration via env var for `MP_MACHINE_TYPE`, with default value from instance types

```suggestion
  MP_MACHINE_TYPE=${MP_MACHINE_TYPE:-$(tail -n 1 "${SHARED_DIR}/instance-types.txt")}
```

wdyt ?

Reply:
It is special for the localzone. It depends on the zone in the step `aws-provision-localzone-subnet` so we can not specify the MP_MACHINE_TYPE randomly. If in the future, we have another way to prepare the localzone subnet, let use update the code with the new achievement.